cmd/xl-v1-multipart.go

```diff
@@ -727,16 +722,12 @@ func (xl xlObjects) PutObjectPart(bucket, object, uploadID string, partID int, s
 	// Add the current part.
 	xlMeta.AddObjectPart(partID, partSuffix, newMD5Hex, size)
 
-	for index, disk := range onlineDisks {
-		if disk == nil {
-			continue
+	for i, disk := range onlineDisks {
+		if disk != OfflineDisk {
```

Comment:
For multiple conditions we can use switch but the previous idea was ignore all the cases we are not going to handle.  But either way is fine.. 

```
if cond1 {
    continue
}
... do something
```

v/s 

```
if !cond {
    ... do something .. 
}
```